cmd/bootstrap/cmd/keygen.go

```diff
@@ -57,10 +85,12 @@ func init() {
 	rootCmd.AddCommand(keygenCmd)
 
 	// required parameters
-	keygenCmd.Flags().
-		StringVar(&flagConfig, "config", "node-config.json", "path to a JSON file containing multiple node configurations (Role, Address, Stake)")
+	keygenCmd.Flags().StringVar(&flagConfig, "config", "node-config.json", "path to a JSON file containing multiple node configurations (Role, Address, Stake)")
 	_ = keygenCmd.MarkFlagRequired("config")
 
+	// optional parameters, used for generating machine account files
+	keygenCmd.Flags().BoolVar(&flagDefaultMachineAccount, "machine-account", false, "whether or not to generate a default (same as networking key) machine account key file")
```

Comment:
not sure I follow this - 

> (same as networking key)

do you imply that key which uses the same encryption type as the networking key?

Reply:
We re-use the networking key for the machine account key for test networks (`localnet` and `benchnet`).